docs: Update FAQ for Ledger derivation paths (#57)
oasisprotocol/docs: docs/general/manage-tokens/faq.md

### Which derivation path should I use on Ledger? ADR 0008 or Legacy? {#ledger-derivation-paths}

To convert your mnemonic phrase into a private key for signing trasactions,
each wallet (hardware or software) performs a *key derivation*. The Oasis
Protocol Foundation standardized the key derivation for official Oasis wallets
in a document called [ADR 0008] back in January 2021. However, the Ledger
hardware wallet already supported signing transactions at that time using a
custom (we now call it *legacy*) derivation path which is incompatible with
the one defined in ADR 0008. Later, in Oasis app for Ledger v2.3.1 support for
ADR 0008 was added so the wallet can request either derivation from the Ledger
device.

The key derivation path defined in ADR 0008 has the following advantages
compared to the legacy one:

- Derivation path is shorter which results in approximately twice as fast
  key derivation (and transaction signing) without compromising security.
- In case your Ledger device is broken or lost and you are unable to retrieve
  a new one, you will be able to import your Ledger mnemonic and restore your
  private key in any Oasis wallet which implements ADR 0008.

> For reasons above we recommend the usage of ADR 0008. However, since there are

Reviewer: ```suggestion
For reasons above, we recommend the usage of ADR 0008. However, since there are
```

no security considerations at stake, Oasis wallets will support legacy

> derivation on Ledger in the foreseeable future.

Reviewer: ```suggestion
derivation on Ledger for the foreseeable future.
```

:::danger

If you happen to import your Ledger mnemonic to a software wallet, consider

> that mnemonic compromised from the hardware wallet standard. If you use a new

Reviewer: ```suggestion
that mnemonic *potentially exposed/compromised*, i.e. not appropriate for a hardware wallet mnemonic anymore. If you use a new
```
